### DISCUSSION TITLE
more VHS artifacts

### DISCUSSION THREAD

**Discussion by grimdraxQZSTN** [ntsc-rs/ntsc-rs#31] [Ideas]
no other VHS effect (except for rarevision maybe) has this. so basically, idk how to describe it, but on a tape that was basically either eaten (by a VCR, _NOT_ by a real human ofc) or is extremely damaged, occasionally (or most of the time), there will be these "crinkles" (i think thats what they are actually called), basically its just a bunch of rainbow colors and static moving down the screen or something, it also at the top and bottom edges seem to smoothly transition with the tape video (IIRC), i really want this feature in so please add it in

also, on some tapes, the top section of the screen will be warped to the left, and i want that too so maybe add that in too

if you can code all of these, you can add these, and i just made a github account just to create this, so thanks ig

**Comment by Degamisu** [comment]
> also, on some tapes, the top section of the screen will be warped to the left,

I second this, however this (skewing) might already be a thing with the `Head Switching` setting

**Reply by grimdraxQZSTN** [reply]
you actually can't change the full on position of the head switching one as it really only stays at the bottom. you can move it a little bit but thats as far as it can go

**Reply by Degamisu** [reply]
yeah. the original NTSC-qt had this feature when turning off `ntsc` video mode.

**Reply by smfthe1199** [reply]
@grimdraxQZSTN I actually understood a bit of your message, but the rest are not. That's `Tracking Noise` w/ `Head Switching` you're talking about, or something. We alrdy have that in ntsc-rs, or if I am wrong correct me!
